README.md

```diff
@@ -72,14 +72,14 @@ Schema for the public configuration file looks like this:
 
```

Comment:
Update this line as well:
"This documentation is current for version 2.0.6 and above."